src/CaptureService/CMakeLists.txt

```diff
@@ -12,12 +12,14 @@ target_include_directories(CaptureService PRIVATE
         ${CMAKE_CURRENT_LIST_DIR})
 
 target_sources(CaptureService PUBLIC
-        include/CaptureService/CaptureService.h
+        include/CaptureService/Capturer.h
+        include/CaptureService/CaptureServiceUtils.h
```

Comment:
Not sure it will fit in one of these changes, but we have this open bug: b/204595539. Now that the `CaptureService` module is even more detached from the gRPC `CaptureService` service (and that the `CaptureService` class is gone), there's even a stronger reason to rename the module to something else to resolve the homonymy.

Reply:
Thanks for the suggestion and also thanks for linking me to the above open bug!

I will rename the module to `CaptureServiceBase`. Also, WDYT if I also rename the `Capturer` to `CaptureServiceBase`? I think the name `CaptureServiceBase` is more readable.

Reply:
> Also, WDYT if I also rename the Capturer to CaptureServiceBase?

I also like it better!